### DISCUSSION TITLE
Winget request vor a tow char region ID, how to setup it?

### DISCUSSION THREAD

**Discussion by Mannshoch** [microsoft/winget-cli#2131] [Q&A] [answered]
If I update winget on a certain PC I get a text that say I have to send a region.
I'm curious, could I setup a specific region in the settings?

**Comment by florelis** [comment, marked as answer]
The region that winget sends is obtained from Windows, it is not configurable with a setting. It may be possible to change the region at the OS level, but I'm not sure if that is possible or advisable.

**Reply by Mannshoch** [reply]
Not sure if I'm able to modify such settings here to modify the system.
